README.md

```diff
@@ -49,7 +49,7 @@ You can see a (customized) instance of Strapping in action by visiting http://oV
 2. Clone the repository:
 
    ```
-   git clone https://github.com/OSAS/strapping-mediawiki strapping
+   git clone https://github.com/axxie/strapping-mediawiki strapping
```

Comment:
Hm...., that's strange.
I edited the README.md after creating pull request. I thought the change will not be included.

Anyway, no, I definitely don't want to change that line in your branch.